### DISCUSSION TITLE
news/20231202/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#288] [Website Discussions]
# news/20231202/

Since its release yesterday, we've been busy testing and exploring Final Cut Pro 10.7.

https://fcp.cafe/news/20231202/

**Comment by RSK-FCP** [comment]
FYI: I doubt there will ever be a button for the scrolling since there is a shortcut. And you CAN already scroll manually any way you like and have the timeline update as you go: https://youtu.be/gu2wdII-yKk 😉

**Reply by latenitefilms** [reply]
I still think once they can finally dump Intel hardware, and every Apple Silicon Mac can handle the scrolling timeline, then they'll "turn on" rendering thumbnails and waveforms during playback by default. At that point, it wouldn't surprise me if they added a scrolling timeline button to the toolbar. I honestly think they're trying to hide the feature a bit currently, so that it's more of a "professional" feature, to avoid all the angry support emails from people trying to use FCPX on base-level Intel hardware.

**Reply by RSK-FCP** [reply]
Agree to disagree. Considering the number of buttons they REMOVED from the GUI with 10.3 alone, I don't see them adding a new one for, again, something that can easily be turned on/off with a shortcut if need be. Also because if you know how to use it, will you even ever turn it off? I have yet to. So a button would be superfluous visual clutter imho. I'd say buttons are reserved for things you _actually_ need **regularly**, not just "maybe". This isn't PPro we're talking about here. 😜

But yes, I think something like that (refresh at all times) is something likely to come with the "iPad Catchup" rebuild. ;)
